tests/unit/testExpressInterface.js

```diff
@@ -14,54 +14,42 @@
  * limitations under the License.
  */
 
-var test = require('tape');
 var lodash = require('lodash');
+var assert = require('assert');
 var merge = lodash.merge;
 var expressInterface = require('../../src/interfaces/express.js');
 var ErrorMessage = require('../../src/classes/error-message.js');
 var Fuzzer = require('../../utils/fuzzer.js');
 var Configuration = require('../fixtures/configuration.js');
 var createLogger = require('../../src/logger.js');
 
-test(
-  "Given invalid, variable input the express interface handler setup should not throw errors"
-  , function ( t ) {
-
-    var f = new Fuzzer();
-
-    t.doesNotThrow(
-      function ( ) {
-
-        f.fuzzFunctionForTypes(
-          expressInterface
-          , ["object", "object"]
+describe('expressInterface', function () {
+  describe('Exception handling', function () {
+    describe('Given invalid input', function () {
+      it('Should not throw errors', function () {
+        var f = new Fuzzer();
+        assert.doesNotThrow(
+          function () {
+            f.fuzzFunctionForTypes(
+              expressInterface
+              , ["object", "object"]
+            );
+            return;
+          }
         );
-
-        return ;
-      }
-      , undefined
-      , "The express interface handler setup should not throw when given invalid types"
-    );
-
-    t.end();
-  }
-);
-
-test(
-  [
-    "Given valid setup variables from the handler setup, the bound express"
-    , "error handler should produce a determinate error message and callback to"
-    , "the given callback function"
-  ].join(" ")
-  , function ( t ) {
-
+      });
+    });
+  });
+  describe('Intended behaviour', function () {
```

Comment:
It is part of the higher level expressInterface describe, but it is not testing exceptionHandling it is exercising the intended success path. I will put the last assert in an it statement though -- thanks for the catch. I don't agree that the its in the sendError and next functions are unnecessary. Part of this test is to see whether the express interface calls the correct functions to pass output to any other plugins and whether it actually attempts to send the error.